src/base/static/components/organisms/site-header.js

```diff
@@ -297,29 +264,27 @@ class SiteHeader extends Component {
           )}
         </LogoTitleWrapper>
         <NavContainer isHeaderExpanded={this.state.isHeaderExpanded}>
-          {this.props.navBarConfig
-            .filter(navBarItem => !navBarItem.hide_from_top_bar)
-            .map((navBarItem, i) => {
-              const NavItemComponent = navItemMappings[navBarItem.type];
-              return (
-                <NavItemComponent
-                  key={i}
-                  position={i}
-                  navBarItem={navBarItem}
-                  currentTemplate={this.props.currentTemplate}
-                  setLeftSidebarComponent={this.props.setLeftSidebarComponent}
-                  setLeftSidebarExpanded={this.props.setLeftSidebarExpanded}
-                  isLeftSidebarExpanded={this.props.isLeftSidebarExpanded}
-                  onClick={() => {
-                    this.setState({
-                      isHeaderExpanded: false,
-                    });
-                  }}
-                >
-                  {navBarItem.title}
-                </NavItemComponent>
-              );
-            })}
+          {this.props.navBarConfig.map((navBarItem, i) => {
```

Comment:
I remove the `.filter(navBarItem => !navBarItem.hide_from_top_bar)` step here because we are no longer using it, and I don't see a realistic need for when we'll have a nav button that is hidden.

Reply:
Good catch-- `hide_from_top_bar` is definitely not needed any more.